docs/workforce-management-framework/topics/con-production-checklist.adoc

```diff
@@ -51,8 +50,8 @@ Most important values
 
 Apache Cordova configuration needs to be adjusted before doing release.
 Configuration can be found in link:{WFM-RC-MobileURL}{WFM-RC-Release-Tag}/src/config.xml[Cordova config.xml file]
-Developers should review cordova file by removing permissions and plugins that aren't used.
-Additionally developers should restrict and remove any origins that aren't used in the application
+Developers should review cordova file by removing permissions and plugins that are not used.
```

Comment:
caps ```Cordova``` 
Developers should review Cordova file by removing permissions and plugins that are not used.